pom.xml

```diff
@@ -155,6 +160,18 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>3.3.0</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.3.1</version>
+                <reportSets>
+                    <reportSet>
+                        <reports>
+                            <report>checkstyle</report>
+                        </reports>
+                    </reportSet>
+                </reportSets>
+            </plugin>
```

Comment:
You will need some additional configuration to make sure that this runs as part of the maven lifecycle.

You can also optionally add a configuration file to modify what rules you do/don't want to enforce

Reply:
I've added configuration to make sure that now runs as part of the maven lifecycle, I think. I've opted not to add the custom configuration file, I've used the google_checks.xml instead.

Reply:
Have you pushed the `google_checks.xml` file to the repo?